### ISSUE TITLE
Calculate percentage for Nimbledroid sites should be based on actual difference rather than including target

### ISSUE THREAD

**Issue by armenzg** [mozilla-frontend-infra/firefox-health-dashboard#186]
Right now the percentage factors a 20% difference.

For instance, given these values: GV: 12s, Chrome: 10s

- Actual percentage: 0%
- Expected percentage: -20%

The colouring will still take the 20% factor.

**Comment by vickychin** [comment]
Yes so in the above example, the table will show -20% coloured yellow.

**Comment by Cheap-Skate** [comment]
I filed another on Bugzilla, I should have posted here.

https://bugzilla.mozilla.org/show_bug.cgi?id=1498832#c3

Hope you find it helpful!
